src/reduxIntegration/asyncOperationReducer.js

```diff
@@ -1,12 +1,15 @@
 import { includes } from 'lodash';
 
-import { getStateForOperationAfterStep } from '../asyncOperationManagerUtils';
+import {
+  getStateForOperationAfterStep,
+  getAsyncOperationsManagerState,
+} from '../asyncOperationManagerUtils';
 
 import {
   ASYNC_OPERATION_STEPS,
 } from '../constants';
 
-function asyncOperationReducer(state = {}, action) {
+function asyncOperationReducer(state = getAsyncOperationsManagerState(), action) {
```

Comment:
For some reason babel transform is hoisting variable declarations out-of-order which was messing up getAsyncOperation as the logic inside was order dependent.  To fix that we needed to have initial state have the two keys: descriptors and operations so here I am just defaulting initial state reducer state to the initial AOM state to fix that.  It should have been as such anyway.